numpyro/distributions/transforms.py

```diff
@@ -1636,3 +1636,29 @@ def _transform_to_simplex(constraint):
 @biject_to.register(constraints.zero_sum)
 def _transform_to_zero_sum(constraint):
     return ZeroSumTransform(constraint.event_dim)
+
+
+@biject_to.register(constraints.union_of_closed_intervals)
+@biject_to.register(constraints.intersection_of_closed_intervals)
+def _transform_to_union_of_intervals(constraint):
+    return ComposeTransform(
+        [
+            ReshapeTransform(
+                forward_shape=constraint.lower_bounds.shape,
+                inverse_shape=(constraint.lower_bounds.size,),
+            ),
+            ExpTransform(),
```

Comment:
I don't understand why this transform is bijective. Could you clarify?

Reply:
I wrote this code snippet to test the constraints of the system. This was the best example I could come up with based on a quick review of the test cases. Please note that this code is not based on any specific logic, so feel free to correct me if I'm wrong or suggest improvements!

Reply:
I see. Probably the tests pass because the domain is quite loose. Exp transform has positive codomain, not union of intervals.